### PR TITLE
Add Tooltip to FA Icons in Simulator

### PR DESCRIPTION
**Pull request by Shivansh2407**
Fixes https://github.com/CircuitVerse/CircuitVerse/pull/1072#issuecomment-596218306

#### Describe the changes you have made in this PR -
I have added Tooltip to the Font Awesome Icons Present in the Simulator (Also to the Edit Layout Icons).

### Screenshots of the changes (If any) -
patch-29

### CODE REVIEW

**Comment by Shivansh2407** [comment]
@satu0king Done 👍 . Please review and merge .

File: app/views/simulator/edit.html.erb [hunk 1]
```diff
@@ -132,22 +132,22 @@
           </div>
           <div style="position:absolute; right:60px; top:30px; z-index:100">
             <button type="button" class="btn zoomButton btn-lg" onclick="globalScope.centerFocus(false)">
-              <i class="fa fa-arrows" aria-hidden="true"></i>
+              <span class="fa fa-arrows" aria-hidden="true" title="Center Focus"></span>
```
Comment (satu0king) [inline app/views/simulator/edit.html.erb]: Can you rename `center focus` it to `zoom to fit` or `center canvas` . Also not sure why its not working for me 
image

**Comment by satu0king** [comment]
Add the titles to `embed.html.erb`

**Comment by Shivansh2407** [comment]
Done @satu0king 👍

**Comment by satu0king** [comment]
I guess it doesn't show up on mac.

**Pull request merged**